### ISSUE TITLE
Bug when downloading pdf with weird filename

### ISSUE THREAD

**Issue by wngr** [grindsa/dkb-robo#61]
Hello!
Thanks (again :-)) for your great library, really helpful for me. Encountered a small issue when downloading the contents of the post box:
```
    with open(f'{directories[1]}/{document["filename"]}', 'wb') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'foo/exAnteCostInformation/Kosteninformation zu Wertpapier BIONTECH SE NAM.-AKT.(SP.ADRS)1/O.N. vom 30.09.2024, 11:51 zu Depot XXXX.pdf
```

I suspect it's because the filename is not properly escaped, specifically the `/` in there.

**Comment by grindsa** [comment]
Let me try to replicate the issue and work on a fix. Just to avoid misunderstandings; the filename of your document is ``Kosteninformation zu Wertpapier BIONTECH SE NAM.-AKT.(SP.ADRS)1/O.N. vom 30.09.2024, 11:51 zu Depot XXXX.pdf`` right?

**Comment by grindsa** [comment]
I pushed some changes into a [new branch](https://github.com/grindsa/dkb-robo/tree/file_name) which should address your issue. Feel free to try and provide feedback....

**Comment by wngr** [comment]
Great, just tried it. Works!

Thanks for being so responsive.

**Comment by grindsa** [comment]
fix got included in [v0.27.1](https://github.com/grindsa/dkb-robo/releases/tag/0.27.1). Hence, I am closing the case.
